packages/terra-embedded-content-consumer/src/terra-dev-site/doc/embedded-content-consumer/EmbeddedContentConsumer.1.doc.jsx

```diff
@@ -32,35 +28,25 @@ const DocPage = () => (
     srcPath={`https://github.com/cerner/terra-framework/tree/master/packages/${name}`}
     examples={[
       {
-        title: 'Basic Embedded Content',
+        title: 'Basic Embedded Content Consumer',
         example: <BasicConsumer />,
         source: BasicConsumerSrc,
       },
       {
-        title: 'Filled Embedded Content',
-        example: <FillConsumer />,
-        source: FillConsumerSrc,
+        title: 'Embedded Content Consumer Lifecycle Statuses',
+        example: <DataStatusConsumer />,
+        source: DataStatusConsumerSrc,
       },
       {
-        title: 'Custom Event Embedded Content',
+        title: 'Embedded Content Consumer that Listens for a Custom Event',
         example: <CustomEventConsumer />,
         source: CustomEventConsumerSrc,
       },
       {
-        title: 'Custom Events Embedded Content',
+        title: 'Embedded Content Consumer with Seamless Communication',
         example: <CustomEventsConsumer />,
         source: CustomEventsConsumerSrc,
       },
-      {
-        title: 'Data Status Embedded Content',
-        example: <DataStatusConsumer />,
-        source: DataStatusConsumerSrc,
-      },
-      {
-        title: 'On Ready Embedded Content',
```

Comment:
This example was actually demonstrating the XFC Provider onReady callback which is not handled by embedded-content-consumer. I removed this example since it was out of scope and demonstrating XFC provider behavior.